src/ct/stylus/arb_node_utils.nim

```diff
@@ -0,0 +1,101 @@
+import std/[httpclient, json, strformat, strutils, sequtils, parseutils, times, sets, os]
+
+const DEFAULT_NODE_URL* = "http://localhost:8547"
+
+# TODO: get name from config? Maybe use SQLite?
+const CONTRACT_WASM_PATH* = getHomeDir() / ".local" / "share" / "codetracer" / "contract-debug-wasm"
+const EVM_TRACE_DIR_PATH* = getTempDir() / "codetracer"
+
+proc jsonRpcRequest(methodParam: string, params: JsonNode): JsonNode =
```

Comment:
Think about error checking in this module.
The code should either have `{.raises: .}` annotations to make it clear how it can fail or it should handle all possible exceptions and translate them to `Result` return value.